scripts/verifyTypescript.ts

```diff
@@ -0,0 +1,209 @@
+import fs from 'fs';
+import ts from 'typescript';
+
+const VERBOSE = !!process.env.VERBOSE;
+
+const TYPESCRIPT_BLOCK = /```(typescript|ts).*\n(?<code>(.|\n)*?)```/;
+
+const HEADER = `
+import { interpret, createMachine, assign, StateMachine, State, SCXML } from 'xstate';
+`;
+
+const USER_MODEL = `
+import { createModel } from 'xstate/lib/model';
+
+const userModel = createModel(
+  // Initial context
+  {
+    name: 'David',
+    age: 30
+  },
+  {
+    // Event creators
+    events: {
+      updateName: (name: string) => ({ name }),
+      updateAge: (age: number) => ({ age }),
+      anotherEvent: () => ({}) // no payload
+    }
+  }
+);
+`;
+
+const BAIL_EXPRESSIONS = [
+  '@xstate/inspect' // xstate-inspect import breaks because of non-browser context?kjj
```

Comment:
I had that damn `non-browser context?kjj`. Always ruining my day! 😆

Reply:
Spent so much time on this before finally going "Fuck it, BAIL" :D

Reply:
I'll just `jj` my way outta here!